postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -432,6 +436,7 @@ def _collect_plan_for_statement(self, row):
                 "db": {
                     "instance": row.get('datname', None),
                     "plan": {"definition": obfuscated_plan, "cost": plan_cost, "signature": plan_signature},
+                    "statement_truncated": self._is_statement_truncated(max_query_size, row),
```

Comment:
Very very minor, but I'm just wondering if there is a reason you put this here instead of the line below "statement"

Reply:
No reason. I agree it would look better below.